packages/bootstrap/scss/core/color-system/_swatch.scss

```diff
@@ -48,11 +48,11 @@ $_default-colors: (
     tertiary-subtle: map.get( $kendo-palette-indigo, 2 ),
     tertiary-subtle-hover: map.get( $kendo-palette-indigo, 3 ),
     tertiary-subtle-active: map.get( $kendo-palette-indigo, 4 ),
-    tertiary: map.get( $kendo-palette-indigo, 8 ),
-    tertiary-hover: map.get( $kendo-palette-indigo, 9 ),
-    tertiary-active: map.get( $kendo-palette-indigo, 10 ),
+    tertiary: map.get( $kendo-palette-indigo, 9 ),
```

Comment:
I might be wrong, but according to the [Figma file](https://www.figma.com/design/YjhpzSsYaZI5VOaamsvIku/Telerik---Kendo-UI-Kit-for-Bootstrap-3.4--UNPUBLISHED-?node-id=12772-8391&m=dev), there should be changes only in tertiary-active and tertiary-on-subtle?